src/Components/Books/Books.js

```diff
@@ -20,9 +21,36 @@ class Books extends Component {
       .catch(error => alert(error.message))
   }
 
+  changeButtonStyling(id) {
```

Comment:
This works. We could consider making the book component a class component with state that holds whether or not the book is added to the list. Then we wouldn't have to use classList.contains to determine if a book is added to the list. But I'm not sure which way would be better.

Reply:
Great work, Jake! My only thoughts are using a toggle to reduce a line of code.